plugins/CMakeLists.txt

```diff
@@ -76,9 +76,9 @@ IF("${PLUGIN_LIST}" STREQUAL "")
 		stereo_enhancer
 		stereo_matrix
 		stk
-		vst_base
-		vestige
-		VstEffect
+		#vst_base
+		#vestige
+		#VstEffect
```

Comment:
It looks like these changes were added by mistake. It will remove anything VST and GigPlayer for everyone.

Reply:
i didn't see apt-get or dnf instructions how to install GigPlayer.

i think it is fine to comment out the VST stuff, because it depends on fixes to wine that are still upstream and not in the Fedora repos.

The master branch needs to compile on Linux with just a few standard commands.  Editing the CMake is bullshit.
Other operating systems can come second, Linux needs to be first class, after all this project is called:  LinuxMultiMediaStudio

Reply:
> i didn't see apt-get or dnf instructions how to install GigPlayer.

The name of the package in Debian is libgig. I don't know why it's not in Fedora but libgig comes from the linuxsampler project and they don't use a license that all projects can work with/around and I've seen Fedora remove stuff before. Our own sample library for instance. There should be a repo for it somewhere out there. To turn gigasampler off hit cmake with this:
```-DWANT_GIG=OFF```

> i think it is fine to comment out the VST stuff, because it depends on fixes to wine that are still upstream and not in the Fedora repos.

The solution for now is to either revert wine to an earlier version or turn VST stuff off with
```-DWANT_VST```

Please revert your changes to plugins/CMakeLists.txt.

Reply:
i didn't want to install wine in the first place, and i'm not going to try to downgrade it.

The CMake script could be changed to look for wine, and if not installed automatically set WANT_VST to off.  CMake could call out to `lsb_release -is` which on my machine returns `Fedora`.  Then we can have simple and standard build commands, that works for all the top linux distros.

Universal build commands for Linux
```bash
cd lmms
mkdir build
cd build
cmake ../.
make
```

Reply:
> The CMake script could be changed to look for wine, and if not installed automatically set WANT_VST to off. 

This can and should be the case but is out of scope of this PR.  Please adjust it accordingly.